src/Symfony/Component/HttpFoundation/Session/Storage/MockArraySessionStorage.php

```diff
@@ -75,8 +73,6 @@ public function __construct($name = 'MOCKSESSID', MetadataBag $metaBag = null)
     }
 
     /**
-     * Sets the session data.
```

Comment:
As the [commit](https://github.com/symfony/symfony/pull/22139/commits/c45aab6b6d9b9c3d595f100937c31fb42cc239f3) message says:

> removed not productive phpdoc title

The title duplicates what's being said in the name of the function. It doesn't add any value to the developer.